libraries/helpers.rb

```diff
@@ -16,14 +16,119 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-module PdnsResource
+module Pdns
+  # Common helper for PowerDNS cookbook
   module Helpers
     def default_user_attributes
       case node['platform_family']
       when 'debian'
-        { home: '/var/spool/powerdns', shell: '/bin/false' }
+        Mash.new(home: '/var/spool/powerdns', shell: '/bin/false')
```

Comment:
Just for people to use symbol as well as string but it is more cosmetic then anything else